### PR TITLE
Run H2 fix test in parallel

### PR DESCRIPTION
**Pull request by m4ksio**
30-50 seconds saved.

### CODE REVIEW

File: node/src/test/kotlin/net/corda/node/internal/AbstractNodeTests.kt [hunk 1]
```diff
@@ -42,12 +42,25 @@ class AbstractNodeTests {
 
     @Test
     fun `H2 fix is applied`() {
-        repeat(if (relaxedThoroughness) 1 else 100) {
+
```
Comment (andr3ej) [inline node/src/test/kotlin/net/corda/node/internal/AbstractNodeTests.kt]: remove all blank lines, the coding standard is vertically dense

File: node/src/test/kotlin/net/corda/node/internal/AbstractNodeTests.kt [hunk 2]
```diff
@@ -42,12 +42,25 @@ class AbstractNodeTests {
...
+        val pool = Executors.newFixedThreadPool(10)
```
Comment (andr3ej) [inline node/src/test/kotlin/net/corda/node/internal/AbstractNodeTests.kt]: this pool is never shutdown

**Comment by m4ksio** [comment]
I have successfully trigger the bug - by commenting the H2 fix and running the test a few times. It's true, that as the original it does not always manifest.

**Comment by andr3ej** [comment]
it shouldn't, but it somehow does. i would like to see proof that the bug is sufficiently reliably triggered in TC before this is merged to master

File: node/src/test/kotlin/net/corda/node/internal/AbstractNodeTests.kt [hunk 3]
```diff
@@ -42,12 +42,22 @@ class AbstractNodeTests {
 
     @Test
     fun `H2 fix is applied`() {
-        repeat(if (relaxedThoroughness) 1 else 100) {
+        val pool = Executors.newFixedThreadPool(5)
+        val runs = if (relaxedThoroughness) 1 else 100
+        (0 until runs).map {
             // Two "nodes" seems to be the magic number to reproduce the problem:
```
Comment (andr3ej) [inline node/src/test/kotlin/net/corda/node/internal/AbstractNodeTests.kt]: this comment needs updating

**Pull request merged**